project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.5.8
```

Comment:
should this downgrade be done separately, if it is actually needed?

Reply:
not a perfect commit but let's take them together - every check for a possible release workflow fix takes about 1h.
if it will help - that's good.
anyway we will have to find out how to upgrade it the latest in future.

Reply:
I upgraded to 1.6.2 for this PR while I'm re-using an older 1.5.8 sbt shell locally for edit/test.